website/docs/auth/service_principal_client_secret.html.markdown

```diff
@@ -111,41 +117,98 @@ $ az vm list-sizes --location westus
 $ az account list-locations
 ```
 
+Finally, since we're logged into the Azure CLI as a Service Principal we recommend logging out of the Azure CLI (but you can instead log in using your user account):
```

Comment:
If we are just going to tell the user to log out as the Service Principal, why did we tell them to log in as the Service Principal in the first place? I find this part a little confusing as well "(but you can instead log in using your user account)"...

Reply:
this exists to validate the credentials being used have permissions to the subscription before moving onto the next step (configuring those credentials in Terraform). We used to have a lot of issues opened where the Service Principal didn't have permissions to the subscription, adding this step in mostly resolved these tbh